engine/gamesys/src/gamesys/components/comp_mesh.cpp

```diff
@@ -396,6 +396,12 @@ namespace dmGameSystem
         return dmGameObject::CREATE_RESULT_OK;
     }
 
+    void* CompMeshGetComponent(const dmGameObject::ComponentGetParams& params)
+    {
+        MeshWorld* world = (MeshWorld*)params.m_World;
+        return (void*)world->m_Components.Get(params.m_UserData);
+    }
```

Comment:
Some components missed their "GetComponent" function, and need to convert from a `dmGameObject::HComponentInternal` handle to an actual pointer.